src/intel/RoomIntel.ts

```diff
@@ -456,7 +458,46 @@ export class RoomIntel {
 			}
 		}
 	}
+	private static autoPoison(room: Room) {
+		
+		if(!Memory.settings.autoPoison.enabled || !room || getAllColonies().length == Game.gcl.level) {
+			return;
+		}
 
+		// poison X rooms at a time
+		let count = 0;
+		for (const colony of getAllColonies()) {
+			if(Game.flags['POISON-'+colony.name]) {
```

Comment:
never use flag names to determine behavior. overmind should be completely agnostic to flag names.